### ISSUE TITLE
Includes are not taken from "VC++ Directories"

### ISSUE THREAD

**Issue by mariusz102102** [Caphyon/clang-power-tools#126]
1. Open attached project
2. Set location to boost in "VC++ Directories" - (in my case this is g:\3rdparty\boost_1_65_1)
3. Compile project with VS - compilation passed
4. Compile with Clang Power Tools- failed:

```
Start Clang Compile

[ INFO ] WILL PROCESS PROJECTS: 
	ConsoleApplication5.vcxproj
[ INFO ] 1. PROCESSING PROJECT C:\Users\Mariusz\documents\visual studio 2017\Projects\ConsoleApplication5\ConsoleApplication5\ConsoleApplication5.vcxproj
clang++.exe: warning: treating 'c-header' input as 'c++-header' when in C++ mode, this behavior is deprecated [-Wdeprecated]
C:\Users\Mariusz\documents\visual studio 2017\Projects\ConsoleApplication5\ConsoleApplication5\stdafx.h:16:10: fatal error: 'boost/lexical_cast.hpp' file not found
#include <boost/lexical_cast.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
Errors encountered during PCH creation

Done Clang Compile
```
Comment:
1. It works fine when path is set in "Additional include directories" g:\3rdparty\boost_1_65_1

[ConsoleApplication5.zip](https://github.com/Caphyon/clang-power-tools/files/1511422/ConsoleApplication5.zip)

**Comment by cpp-red-lion** [comment]
Indeed, Clang Power Tools currently assume **default** configuration for `VC++ Directories`.

We are aware that some users prefer the `<IncludePath>` VS property for setting **Include** directories instead of the more common way of specifying `<AdditionalIncludeDirectories>`.

We'll address this in a future release.
Thanks for the suggestion.

**Comment by hero101111** [comment]
We've addressed this issue as part of #132 
It should work now. 

---

Please note that this fix is currently live only for the github repository.
The latest Visual Studio extension (2.0) does not contain the fix. It will be included in the next release.

Until then, as a workaround, you can manually replace `clang-build.ps1` in the VS extension directory with the latest from github. To locate the extension directory search for `clang-build.ps1` in `%LOCALAPPDATA%`

Don't forget to restart Visual Studio after replacing the script file.
